jsoc/admins.md

```diff
@@ -4,10 +4,9 @@ The JSoC (encompasses GSoC, GSoD, etc.) program admins have a few responsibiliti
 
 The current JSoC admins are: 
 - [Avik Sengupta](https://twitter.com/aviksengupta) - Julia Project Lead at NumFOCUS
-- [Chris Rackauckas](https://twitter.com/ChrisRackauckas)
-    - (Currently leads [SciML under NumFocus](https://sciml.ai/dev/#google_summer_of_code) as well)
-- [Logan Kilpatrick](https://twitter.com/OfficialLoganK) - Julia Language Community Manager
-- [Viral Shah](https://twitter.com/Viral_B_Shah) - Julia co-creator and CEO of JuliaHub
+- [Jacob Scott Zelko](https://jacobzelko.com/) - Prominent contributor to the [JuliaHealth](https://juliahealth.org/) ecosystem
```

Comment:
If it is alright with @DilumAluthge (and all other parties I may not be thinking of), maybe put me here as:

"Current lead of the [JuliaHealth](https://juliahealth.org/) ecosystem"

Otherwise, looks great! Many thanks @LilithHafner :pray:

Reply:
```suggestion
- [Jacob Scott Zelko](https://jacobzelko.com/) - Current lead of the [JuliaHealth](https://juliahealth.org/) ecosystem
```

Thanks for providing a more accurate characterization of your role :)